README.md

```diff
@@ -1,4 +1,6 @@
-# PHPUnit 3.7.x php 5.3-8.0 compatible
+# PHPUnit 3.7.x php 5.3-8.1 compatible
+
+> This project is not compatible with composer v2 which does not support pear repositories anymore. 
```

Comment:
although this applies only to local dev/testing, as pear is included only in `require-dev`. `zf1s/phpunit` may still be installed with composer v2 from another project (namely `zf1s/zf1`) and it will install and work properly.

Reply:
If needed, could build composer repository with v1 that uses static files from github pages:
- https://github.com/eventum/composer